scripts/simapp/setup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -o errexit -o nounset -o pipefail
+set -o errexit -o nounset
```

Comment:
Why this change?

Reply:
```
In ./setup.sh line 2:
set -o errexit -o nounset -o pipefail
                             ^------^ SC2039: In POSIX sh, set option pipefail is undefined.

For more information:
  https://www.shellcheck.net/wiki/SC2039 -- In POSIX sh, set option pipefail ...
```

Reply:
I see. Can we switch from `#!/bin/sh` to `#!/bin/bash` in order to keep the feature?

Reply:
```
docker run --rm -it interchainio/simapp:nightly-2020-08-05 sh
~ # bash
sh: bash: not found
```

Reply:
Ah, the image is based on busybox (low end system for routers and stuff). Alright then, thanks for checking!